src/components/text-editor/test-setup/editor-view-builder.spec.ts

```
@@ -0,0 +1,249 @@
+            originalDocument = global.document;
+        });
+
+        afterEach(() => {
+            // Restore original values
+            global.window = originalWindow;
+            global.document = originalDocument;
+        });
+
+        it('should create mock DOM if none exists', () => {
+            // Temporarily remove window and document
+            delete global.window;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)